rxkeystore/src/androidTest/java/com/nexenio/rxkeystore/provider/asymmetric/rsa/RxRSACryptoProvider2Test.java

```diff
@@ -0,0 +1,75 @@
+package com.nexenio.rxkeystore.provider.asymmetric.rsa;
+
+import com.nexenio.rxkeystore.RxKeyStore;
+import com.nexenio.rxkeystore.provider.BaseCryptoProviderTest;
+import com.nexenio.rxkeystore.provider.RxCryptoProvider;
+import com.nexenio.rxkeystore.provider.asymmetric.RxAsymmetricCryptoProvider;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.util.Locale;
+
+import androidx.annotation.NonNull;
+import androidx.test.platform.app.InstrumentationRegistry;
+import io.reactivex.Completable;
+
+public class RxRSACryptoProvider2Test extends BaseCryptoProviderTest {
```

Comment:
Why didn't you add the tests to `BaseAsymmetricCipherProviderTest`?

Reply:
I needed a different Keystore and `CipherProvider` as the extending classes are using ones that did not work with the Android Keystore.

Reply:
Should be named `RsaCipherProviderTest2` or similar then. Can't you extend `RsaCipherProviderTest` and overwrite the provider / keystore logic?

Reply:
I could but the other tests fail for using android Keystore so there was no point in overriding it and ignoring the tests. Wasn't sure if that is unexpected or not.